draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1411,14 +1384,16 @@ Outputs:
 : entries:
   : An array of objects, each consisting of
 
-    leaf_input:
+    log_entry:
     : The base64 encoded `TransItem` structure of type `x509_entry_v2` or
       `precert_entry_v2` (see {{tree_leaves}}).
 
-    log_entry:
-    : The base64 encoded log entry (see {{log_entries}}). In the case of an
-      `x509_entry_v2` entry, this is the whole `X509ChainEntry`; and in the case
-      of a `precert_entry_v2`, this is the whole `PrecertChainEntryV2`.
+    submission:
+    : JSON object representing the original certificate, or precertificate,
+      submission. In the case of an `x509_entry_v2` entry, the object would
+      contain `certificate` and `chain` fields; ; and in the case
+      of a `precert_entry_v2`, it would contain `precertificate` and `chain`
+      fields.
```

Comment:
Done, after adding the caveat that the JSON objects contains the entire chain, even if the trust anchor was missing from the submission.